src/libs/OptionsListUtils.ts

```diff
@@ -1783,6 +1789,7 @@ function getOptions(
         policyReportFieldOptions = [],
         recentlyUsedPolicyReportFieldOptions = [],
         includeInvoiceRooms = false,
+        isSearchingForReports = false,
```

Comment:
I looked into that but it seems we pass `searchInputValue` in other cases other than when we're searching for reports via the ChatFinderPage [e.g.](https://github.com/Expensify/App/blob/e24cf4d78e7b4dee5a8028e180a75c98bd87e868/src/libs/OptionsListUtils.ts#L2126)

Reply:
Yeah, do we want the same behavior in these other places or no?

Reply:
The only case where it would matter is:
- For some reason a room that `!isSearchableViaParticipant` is returned without participant data and stored into Onyx.
- We display it in those other cases when it shouldn't actually be displayed.

Although this should theoretically never happen as long as our back-end logic is sound.

Reply:
Updated